src/components/CoinListChart/index.js

```diff
@@ -21,6 +21,9 @@ export const CoinListChart = (props) => {
         legend: {
           display: false,
         },
+        tooltips: {
+          enabled: false,
+        },
         scales: {
           xAxes: [
```

Comment:
Did you forget to close the square bracket?

Reply:
They are closed bro